metadata.json

```diff
@@ -6,6 +6,9 @@
   "settings-schema": "org.gnome.shell.extensions.bluetooth-quick-connect",
   "gettext-domain": "bluetooth-quick-connect",
   "shell-version": [
-    "3.38"
+    "3.38",
```

Comment:
It does not work for 3.38. I get following error:
```
The settings of extension bluetooth-quick-connect@bjarosze.gmail.com had an error:
Gtk.BuilderError: /home/bjarosze/.local/share/gnome-shell/extensions/bluetooth-quick-connect@bjarosze.gmail.com/Settings.ui:129:1 Unhandled tag: <layout>

Stack trace:

build@/home/bjarosze/.local/share/gnome-shell/extensions/bluetooth-quick-connect@bjarosze.gmail.com/prefs.js:23:23
buildPrefsWidget@/home/bjarosze/.local/share/gnome-shell/extensions/bluetooth-quick-connect@bjarosze.gmail.com/prefs.js:83:27
_init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:207:40
OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:122:28
asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:132:22
run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:177:20
main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
run@resource:///org/gnome/gjs/modules/script/package.js:222:19
start@resource:///org/gnome/gjs/modules/script/package.js:206:8
@/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```

If it can't be easly fixed, maybe just remove `3.38`?

Reply:
Good point, not sure what to do to maintain backward compatibility, maybe we can replicate the .ui file with the previous version and load one or another depending on the version of G-S.

Let me try it